cypress/support/commands/p2p.js

```diff
@@ -1146,7 +1240,7 @@ Cypress.Commands.add(
 
 Cypress.Commands.add(
   'c_addSellOrderDetails',
-  (paymentMethod, amount, rate, min, max) => {
+  (paymentMethod, amount, rate, min, max, rateType) => {
```

Comment:
because now you are passing an object as parameter this also needs to be updated to 
```suggestion
  (options) => {
  const {
  paymentMethod = '',
  amount = '',
  rate='',
  min='',
  max='',
  rateType=''
  }=options
```